lib/rucio/web/rest/flaskapi/v1/locks.py

```diff
@@ -76,18 +78,65 @@ def get(self, scope_name):
         :status 406: Not Acceptable.
         :returns: Line separated list of dictionary with lock information.
         """
+
         did_type = request.args.get('did_type', default=None)
-        if did_type != 'dataset':
+        if did_type not in ('dataset', 'container'):
```

Comment:
don't we have a collection type that can be checked here? or is the string check required?